Optimization.Tests/DeflatedSharpeRatioFitnessTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Optimization.Tests
+{
+
+    [TestFixture]
+    public class DeflatedSharpeRatioFitnessTest
+    {
+
+
+        [Test]
+        public void CalculateExpectedMaximumTest()
+        {
+            var unit = new DeflatedSharpeRatioWrapper();
+            unit.Initialize(null);
+            var actual = unit.CalculateExpectedMaximum();
+
+            Assert.AreEqual(4.2894064662728244, actual, 0.0000001);
```

Comment:
~0.1132 is not the expected maximum, is the SR_0, if you check the equation (1) in page 7, the Expected maximum is the mean SR plus the annualized SR_0